cypress/e2e/ui/menu.cy.js

```diff
@@ -5,64 +5,491 @@ describe('Menu', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Requests').expect_show_list_title('Requests');
+        });
+      });
+    });
+
+    describe('Control >', () => {
+      it('Policy Profiles', () => {
+        cy.menu('Control', 'Policy Profiles').expect_show_list_title('Policy Profiles');
+      });
+
+      it('Policies', () => {
+        cy.menu('Control', 'Policies').expect_show_list_title('Policies');
+      });
+
+      it('Events', () => {
+        cy.menu('Control', 'Events').expect_show_list_title('Events');
+      });
+
+      it('Conditions', () => {
+        cy.menu('Control', 'Conditions').expect_show_list_title('Conditions');
+      });
+
+      it('Actions', () => {
+        cy.menu('Control', 'Actions').expect_show_list_title('Actions');
+      });
+
+      it('Alert Profiles', () => {
+        cy.menu('Control', 'Alert Profiles').expect_show_list_title('Alert Profiles');
+      });
+
+      it('Alerts', () => {
+        cy.menu('Control', 'Alerts').expect_show_list_title('Alerts');
+      });
+
+      it('Simulation', () => {
+        // cy.menu('Control', 'Simulation').expect_show_list_title('');
+      });
+
+      it('Import / Export ', () => {
+        cy.menu('Control', 'Import / Export').expect_show_list_title('Import / Export');
+      });
+
+      it('Log', () => {
+        cy.menu('Control', 'Log').expect_show_list_title('Log');
+      });
+    });
+
+    describe('Settings >', () => {
+      it('My Settings ', () => {
+        cy.menu('Settings', 'My Settings').get('#main-content h3').contains('General');
+      });
+
+      it('Application Settings', () => {
+        cy.menu('Settings', 'Application Settings').get('#control_settings_accord > .panel-title');
+      });
+
+      it('Tasks', () => {
+        cy.menu('Settings', 'Tasks').get('#main-content a').contains('My Tasks');
+      });
+
+      it('Documentation', () => {
+        const primary = '#main-menu nav.primary';
+        const secondary = 'div[role="presentation"] > .bx--side-nav__items';
+
+        cy.get(`${primary} > ul > li`)
+          .contains('a > span', 'Settings')
+          .click();
+
+        cy.get(`${secondary} > li`)
+          .contains('a > span', 'Documentation').parent().then((a) => {
+            return a[0].href.includes('manageiq.org/docs');
+          });
+      });
+
+      it('ManageIQ.org', () => {
+        const primary = '#main-menu nav.primary';
+        const secondary = 'div[role="presentation"] > .bx--side-nav__items';
+
+        cy.get(`${primary} > ul > li`)
+          .contains('a > span', 'Settings')
+          .click();
+
+        cy.get(`${secondary} > li`)
+          .contains('a > span', 'ManageIQ.org').parent().then((a) => {
+            return a[0].href.includes('manageiq.org');
+          });
+      });
+
+      it('About', () => {
+        cy.menu('Settings', 'About').then(() => {
+          cy.wait(50000); // Need to wait before getting the modal container or else the test fails
```

Comment:
That's really strange and shouldn't be necessary since `cy.get` waits for the div to be present and visible - It could be that the `.bx--modal-container` actually exists on the page but is unpopulated.  However, I'd expect that it being hidden would pass by `cy.get`.  Can you give more details on why this fails?

Reply:
I believe when I was testing what was happening was that this modal was doing various API calls before being loaded. Without the wait it was trying to find the div for a while and eventually fails before the API calls were finished. With the wait it gives the API calls enough time to finish and then it will look for the div.

Reply:
I thought it was strange as well since other pages don't have this issue, something about this being a modal might be causing it to fail